packages/vuetify/src/composables/group.ts

```diff
@@ -56,17 +81,23 @@ export function useGroupItem (
     return group.isSelected(id)
   })
 
+  const selectedClass = computed(() => isSelected.value && (group.selectedClass.value ?? props.selectedClass))
+
   return {
     isSelected,
     toggle: () => group.select(id, !isSelected.value),
     select: (value: boolean) => group.select(id, value),
+    selectedClass,
+    value,
+    disabled,
   }
 }
 
 export function useGroup (
   props: GroupProps,
   injectKey: InjectionKey<GroupProvide>
 ) {
+  const isUnmounted = ref(false)
   const items = reactive<GroupItem[]>([])
   const selected = useProxiedModel(
```

Comment:
maybe `selected` could be a `Set`? also accepting `Set` as a model would be a nice feature, it doesn't seem to be supported now

Reply:
:+1:, set wasn't reactive in vue 2 but is now.

Reply:
```
Set in => Set out
Array in => Array out
nothing in => what out?
```

Reply:
hmm good question, i think i'd go for array and use `Set` only if dev explicitely uses `Set` in `v-model`

Reply:
I am dropping this. It seems like way more hassle than it's worth. Internally things don't really get much easier or more performant (unless I'm just stupid and missing something obvious). And from a DX point of view, users can still do 
```
:model-value="[...set.values()]" @update:model-value="set = new Set($event)"
```
if they really want to store value in a set.